bin/cli.js

```diff
@@ -71,7 +71,12 @@ For more information, see https://webpack.js.org/api/cli/.`);
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
 			if (err.name !== "ValidationError") {
-				throw err;
+				console.error("\n\u001b[1m\u001b[31mWebpack not found, please install webpack");
+				console.error(
+					"\u001b[1m\u001b[31mAlternatively, run 'webpack(-cli) --help' for usage info.\u001b[39m\u001b[22m\n"
```

Comment:
Merge two `console.error` here using concatenation or template literals

Reply:
Will do that :smile: